### PR TITLE
Enhanced Inventory 2.10b update

### PR DESCRIPTION
**Pull request by Teraunce**
I swear I can stop anytime I want to.

### CODE REVIEW

**Comment by ryan-dyer-sp** [comment]
@Teraunce Doesnt look like this is being maintained anymore.  Would you mind publishing your release on your fork?

**Review by ryan-dyer-sp — COMMENTED** [review]
It looks like your spacing is off.  Guessing original is using tabs and you're using spaces.  Or just the amount of spaces per tabstop is different.  I prefer a 2 space tab as well, but it makes it hard to follow the "xml" when the spacing is off.

Also I admit I am not a modder, but not sure why all the libs got committed.  Guessing just a slip of the commit.

**Comment by Teraunce** [comment]
> @Teraunce Doesnt look like this is being maintained anymore. Would you mind publishing your release on your fork?

My release is broken. I got it to compile but it fails to load due to UI shenanigans I suspect.
